examples/annotated-http-service-kotlin/src/test/kotlin/example/armeria/server/annotated/kotlin/AnnotatedServiceTest.kt

```diff
@@ -53,4 +53,12 @@ class AnnotatedServiceTest {
             assertThat(it.contentUtf8()).isEqualTo("OK")
         }
     }
+
+    @Test
+    fun testNothingReturnType() {
+        val client = client()
+        client.get("/throw/error").aggregate().join().let {
+            assertThat(it.status()).isEqualTo(HttpStatus.INTERNAL_SERVER_ERROR)
+        }
+    }
```

Comment:
Could you move the test cases under https://github.com/line/armeria/tree/e294957e2999d361c3a2450ac85a2b54c1d2322b/kotlin/src/test/kotlin/com/linecorp/armeria/server?

Reply:
Thank you for advice. I moved it.